services/notebooks/notebooks_service.py

```diff
@@ -70,6 +71,12 @@ def decorated(*args, **kwargs):
     return decorated
 
 
+@app.route('/health')
```

Comment:
do we need a `SERVICE_PREFIX` here?

Reply:
no because health checks are run on the container directly